diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -5,7 +5,7 @@
 import pgDependency from 'pg';
 import { fileURLToPath } from 'url';
 import { connect } from './connect.js';
-const filename = fileURLToPath(import.meta.url);
+const filename = fileURLToPath((import.meta.url || 'file:///worker.mjs'));
 export function postgresAdapter(args) {
     const postgresIDType = args.idType || 'serial';
     const payloadIDType = postgresIDType === 'serial' ? 'number' : 'text';
