diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -4,7 +4,7 @@
 import { createDatabaseAdapter, defaultBeginTransaction, findMigrationDir } from 'payload';
 import { fileURLToPath } from 'url';
 import { connect } from './connect.js';
-const filename = fileURLToPath(import.meta.url);
+const filename = fileURLToPath((import.meta.url || 'file:///worker.mjs'));
 export function sqliteAdapter(args) {
     const sqliteIDType = args.idType || 'number';
     const payloadIDType = sqliteIDType === 'uuid' || sqliteIDType === 'uuidv7' ? 'text' : 'number';
