diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -61,7 +61,7 @@
 export { getFieldsToSign } from './auth/getFieldsToSign.js';
 export { getLoginOptions } from './auth/getLoginOptions.js';
 export * from './auth/index.js';
-const filename = fileURLToPath(import.meta.url);
+const filename = fileURLToPath((import.meta.url || 'file:///worker.mjs'));
 const dirname = path.dirname(filename);
 let checkedDependencies = false;
 /**
